pkg/docker/config/config.go

```diff
@@ -318,6 +342,26 @@ func deleteAuthFromCredHelper(credHelper, registry string) error {
 	return helperclient.Erase(p, registry)
 }
 
+func deleteAuthFromCredsStore(credHelper string) error {
+	creds, err := listAuthFromCredHelper(credHelper)
+	if err != nil {
+		return err
+	}
+	for registry := range creds {
+		err = deleteAuthFromCredHelper(credHelper, registry)
```

Comment:
Single line
if err = ...

Reply:
Should we ignore this ENOEXIST type error?

Reply:
I think no. the code before this listAuthFromCredHelper has made sure the authentication of the registry exist